pkg/servicescm/servicescm_test.go

```diff
@@ -161,17 +155,37 @@ func TestValidateDependencies(t *testing.T) {
 			expectedErr: false,
 		},
 		{
-			name: "bootstrap depends on all non-bootstrap services",
+			name: "bootstrap service requires node variable in command",
```

Comment:
The bootstrap service here requires a node variable substitution in it's command, which is invalid. Since we have `expectedErr: true` for this test case, the naming can stay IMO. Let me know if you feel strongly otherwise

Reply:
The confusion is that the test name makes it seem it is valid to have a node variable in the bootstrap command. But I am not going to hold the PR due to this.